src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -257,6 +257,11 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    This target can be removed after source-build picks up version of Arcade that has
```

Comment:
Please link to a tracking issue for this work.

Reply:
Thanks - will fix.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/aaef75a05709138632d2dca8db9def9944f1bacb